### ISSUE TITLE
`rules` doesn't accept `...vitest.configs.recommended.rules`

### ISSUE THREAD

**Issue by alundiak** [vitest-dev/eslint-plugin-vitest#558]
ESLINT v.9.x (meaning I rely on `eslint.config.js`)
NodeJS v18.x

I tried as it's suggested in README:

`import vitest from '@vitest/eslint-plugin';`
...

```js
plugins: {
      vitest
},
rules: {
      ...vitest.configs.recommended.rules,
}
```

But line `...vitest.configs.recommended.rules` in config causes error.
And using `...vitest.configs.all.rules` also is NOT OK.

ERROR:
![bcd6969354eb4e7fd54cceb5b00fcac7b618481e9a213cffcebe37a4b9e18ea7](https://github.com/user-attachments/assets/f58ecff3-e05a-4d64-9114-a07bc8f5ac8f)

**Working solution is ONLY when**:
```js
extends: {
      vitest.configs.recommended
}
```
**YES, WITHOUT vite in plugins**
```js
plugins: {
      vitest
}
```
**YES, WITHOUT**
And I checked - vitest rules applied OK (I deliberatelly caused for example same test names, and I saw error - so OK)

But when I add `vitest` or `'vitest': vitest` to `plugins` 
then `npm run lint` throws an `SyntaxtError: Unexpected identified`.

FYI @veritem 
I don't know if it's bug and rules should be accepted or it's a config feature, when we need ONLY to pass  to `extends`.

**Comment by veritem** [comment]
refer read on how to configure the plugin

**Comment by alundiak** [comment]
But README doesn't reflect it properly, that was reason of this issue.
